site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -155,6 +155,65 @@
             </div>
 
 
+            <div id="manual-grading" class="customization_item">
+                <h2>Manual Grading</h2>
+                <p class="rg_info_message">You may manually assign final grades to specific students. The note is only invisible to the instructor.</p>
```

Comment:
Typo, did you mean 'only visible'?

Reply:
Good catch. Fixed in [964dbf8](https://github.com/Submitty/Submitty/pull/10653/commits/964dbf8e77ae3a5f370f09f9db93b38f726087eb)